core/domain/exp_jobs_one_off.py

```diff
@@ -381,10 +381,17 @@ def map(item):
         commit_log_models = (
             exp_models.ExplorationCommitLogEntryModel.get_all_exploration_commits( # pylint: disable=line-too-long
                 exploration.id, exploration.version))
-        change_lists = [commit.commit_cmds for commit in commit_log_models]
 
         # Create and save state id mapping model for all exploration versions.
-        for exploration, change_list in zip(explorations, change_lists):
+        for exploration, commit in zip(explorations, commit_log_models):
+            if commit is None:
+                yield Exception(
```

Comment:
Hi @prasanna08 -- sorry, turns out this is wrong and I missed it. You can't yield an Exception; you need to yield a tuple. See @pranavsid98's job -- https://github.com/oppia/oppia/pull/4125 -- can you take a look and fix this? Sorry for missing it.

Reply:
OK, I see. I can also follow @pranavsid98's approach and use snapshot models (if they don't have the issue of missing commit_cmds). So what should I do?